components/Table/Table.tsx

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { cn } from '@/utils/utils';
+
+const Table = React.forwardRef<
+  HTMLTableElement,
+  React.HTMLAttributes<HTMLTableElement> & { className?: string }
+>(({ className, ...props }, ref) => (
+  <table
+    ref={ref}
+    className={cn(
+      'w-[1136px] h-[336px] border rounded-[6px] border-datatableborder',
```

Comment:
We should not be using pixels to define width and height. Why are you using custom values here?

The same goes with `rounded-[6px]'. Don't forget to checkout the [tailwind docs](https://tailwindcss.com/docs/border-radius#rounded-corners) if you're unsure if a class exists. 

I'd also recommend the [tailwind intellisense](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) extension if you don't have it already.